guides/source/configuring.md

```diff
@@ -79,6 +79,7 @@ Below are the default values associated with each target version. In cases of co
 - [`config.active_record.run_after_transaction_callbacks_in_order_defined`](#config-active-record-run-after-transaction-callbacks-in-order-defined): `true`
 - [`config.active_record.run_commit_callbacks_on_first_saved_instances_in_transaction`](#config-active-record-run-commit-callbacks-on-first-saved-instances-in-transaction): `false`
 - [`config.active_record.sqlite3_adapter_strict_strings_by_default`](#config-active-record-sqlite3-adapter-strict-strings-by-default): `true`
+- [`config.active_record.generate_secure_token_on`](#config-active-record-generate_secure_token_on): `:initialize`
```

Comment:
Once you've made the addition to `load_defaults`, you'll have to move this up because this list gets checked for being alphabetical